nipst/nipst_test.go

```diff
@@ -58,10 +61,55 @@ func TestNIPSTBuilder(t *testing.T) {
 
 	select {
 	case <-nipstChan:
-	case <-time.After(1 * time.Second):
-		t.Fail()
+	case <-time.After(5 * time.Second):
+		assert.Fail("timeout")
 		return
 	}
 
 	nb.Stop()
 }
+
+func TestNIPSTBuilderWithRPCClients(t *testing.T) {
```

Comment:
need to add a test that actually validates the created nipst

Reply:
I added assertion to the nipst validity, which is merely a mock now. Will rewrite these tests after the upcoming changes.